cmd/argoexec/commands/wait.go

```diff
@@ -85,4 +85,4 @@ func waitContainer() error {
 	}
 
 	return nil
-}
+}
```

Comment:
Very minor, but I think `golang-ci` would have complained about this:

```suggestion
}

```